src/pages/EditMonitor.vue

```diff
@@ -1456,6 +1456,12 @@ message HealthCheckResponse {
             }
             this.monitor.game = newGameObject.keys[0];
         },
+
+        "monitor.ignoreTls"(newVal) {
```

Comment:
What is happening here and where is this called?

The method name is impossible to differentiate from the actual data => knowing when this is called is basically impossible.

Let's use method names like above and below ^^

If this is how vue handles triggers, please use `@input` instead.

Reply:
This will setup a watch for monitor.ignoreTls. Without this watch if a user, enables Certificate Expiry Notification and then opts to Ignore SSL errors, the Certificate-Expiry option will be still be enabled but the user will not be able to modify the option. This watch sets the Certificate-Expiry option to false if Ignore SSL is updated to true.

> Let's use method names like above and below ^^

This pattern has been used in the same file for example to automatically set port when monitor type changes. 
https://github.com/louislam/uptime-kuma/blob/21a01e864146941d37659555c4531dbd9512eaaf/src/pages/EditMonitor.vue#L1375

Should I use @input instead?

Reply:
Sorry, should have looked further up in the file.
I only looked at the method `currentGameObject` which is directly above.

=> reasonable, LGTM